src/addAirTableRow.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+const Airtable = require('airtable');
+const util = require('util');
+
+// The fields that I could match in the example airbase table provided to me
+// by ben:
+// I commented out fields that I couldn't match
+// https://airtable.com/tblQ17oiafuut0mpP/viwvoqxOoTFF0JsdW
+const defaultMap = {
```

Comment:
Awesome, this is what we will have to work with the mapping_form to customize. Same with the input columns. 
So will keep an eye on this.